### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by CoderSheepX** [CVIU-CSU/CR4CACD#2]
Sorry to bother you again. I can't find the  implementation of the GRAM and RRAM is in [roi_attention_head.py] and I don't know how the gram and rram combine together. Is the code complete? I am a little confused. Hope for your answer and forgive my ignorance.

**Comment by SuperJie12138** [comment]
Indeed, the code is incomplete. I have found and uploaded the missing code files from the author's team project. Thank you for discovering this blind spot. Now the code link can be opened successfully.
